tests/affiliatedcertification/affiliatedcerthelper/affiliatedcerthelper.go

```diff
@@ -183,18 +185,32 @@ func updateCsv(namespace string, csv *v1alpha1.ClusterServiceVersion) error {
 	return nil
 }
 
-func DeleteCsv(prefixCsvName string, namespace string) error {
-	csv, err := getCsvByPrefix(prefixCsvName, namespace)
+// Deploys an operator and returns when its deployment is confirmed to exist.
+func DeployAndVerifyOperatorSubscription(operatorPackage, chanel, namespace, group,
+	sourceNamespace, deploymentName string) error {
+	operatorSubscription := utils.DefineSubscription(
+		operatorPackage+"-subscription",
+		namespace,
+		chanel,
+		operatorPackage,
+		group,
+		sourceNamespace)
+
+	err := DeployOperator(namespace, operatorSubscription)
+
 	if err != nil {
-		return err
+		return fmt.Errorf("Error deploying operator "+operatorPackage+": %w", err)
 	}
 
-	err = globalhelper.APIClient.ClusterServiceVersions(namespace).Delete(
-		context.TODO(), csv.GetName(), metav1.DeleteOptions{},
-	)
+	// confirm that operator is installed and ready
+	gomega.Eventually(func() bool {
```

Comment:
Please move Eventually call to tests package.  Let's avoid using gomega API calls in helpers.